LoopKit/DeviceManager/DeviceStatusProgress.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  DeviceStatusProgress.swift
+//  LoopKit
+//
+//  Created by Nathaniel Hamming on 2020-06-30.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public protocol DeviceStatusProgress {
```

Comment:
I believe this progress bar is always related to device lifecycle, right? Does it make sense to have a name that conveys that? Like DeviceLifecycleProgress? There are a whole lot of "Status" fields related to devices (delivery status, connectivity status, etc), to the point that the word "status" doesn't convey much.

Reply:
Just looked again, and I think you are right. Generally, this bar is related to some lifecycle. I'll update as suggested.